### PR TITLE
DOCS: Cleanup guide intro

### PR DESCRIPTION
**Pull request by dwhswenson**
- [x] Add gather CLI command page
- [x] Add command examples on intro page
- [ ] Link to tutorial on making SLURM jobs
- [ ] Add figures for workflows

### CODE REVIEW

File: docs/guide/introduction.rst [hunk 1]
```diff
@@ -66,7 +66,11 @@ The commands used for set up the CLI are:
 * the :ref:`cli_plan-rbfe-network`
 * the :ref:`cli_plan-rhfe-network`
 
-.. TODO add command example here;
+For example, you can create an RBFE network using
```
Comment (IAlibay) [inline docs/guide/introduction.rst:69]: RBFE -> Relative Binding Free Energy (RBFE) first use of a the acronym in this document right?

File: docs/guide/introduction.rst [hunk 2]
```diff
@@ -66,7 +66,11 @@ The commands used for set up the CLI are:
...
+
+.. code:: bash
+
+    $ openfe plan-rbfe-network -p protein.pdb -M dir_with_sdfs/
 
 These will save the alchemical network represented as a JSON file for each
 edge of the :class:`.AlchemicalNetwork` (i.e., each leg of the simulation).
```
Comment (IAlibay) [inline docs/guide/introduction.rst:76]:
```suggestion
edge of the :class:`.AlchemicalNetwork` (i.e., each leg of the alchemical cycle).
```

I think this is what you're trying to say here?

**Review by IAlibay — CHANGES_REQUESTED** [review]

**Comment by IAlibay** [comment]
Just added a minimum sentence structure fix ahed of @richardjgowers' possible great merge

**Comment by IAlibay** [comment]
@dwhswenson if you don't mind I'll take over here and address the review comments I made & merge eagerly so we can have semi sensical docs for this week

**Comment by dwhswenson** [comment]
Please do! I've been distracted in other directions.

**Review by richardjgowers — APPROVED** [review]

**Pull request merged**